data/com.usebottles.bottles.metainfo.xml.in

```diff
@@ -8,13 +8,13 @@
   <summary>Run Windows Software</summary>
   <developer_name translatable="no">Bottles Contributors</developer_name>
   <description>
-    <p>Bottles lets you run Windows software on Linux, such as applications and games. It introduces a workflow that helps you organize by categorizing each software to your liking. Bottles provides several tools and integrations to help you manage and optimize your applications.</p>
+    <p>Bottles lets you run Windows software on Linux, such as applications and games. It introduces a workflow that helps you organize by categorizing each software to your liking. Bottles provide several tools and integrations to help you manage and optimize your applications.</p>
```

Comment:
"Bottles" in this context is a proper noun in singular, as it's an app called Bottles. In that case, it should be "provides" and not "provide".

That being said, maybe we should reword this to avoid confusion and giving the impression that it's a typo?

Reply:
For now, I have reverted the change, I am open to rewording it too.

Reply:
Now that I think of it, the first sentence starts with "Bottles", followed by a verb in singular. I'm not sure if we can completely reword it without mentioning "Bottles" without sounding bad

Reply:
I'll keep it as is. I think we've waited long enough.